packages/core/lib/command-utils.js

```diff
@@ -201,6 +203,37 @@ const runCommand = async function (command, options) {
   return await command.run(options);
 };
 
+const processHelpInput = inputStrings => {
+  //When we think user is trying to access help incorrectly, provide them with some helpful information.
+  if (inputStrings.length > 2 && ["help", "--help"].includes(inputStrings[1])) {
```

Comment:
what is the point of doing a check for the `length` property on these `if` conditions?